core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/async/FileAsyncRequestBody.java

```diff
@@ -169,10 +169,11 @@ public FileAsyncRequestBody build() {
         private final Subscriber<? super ByteBuffer> subscriber;
         private final int chunkSize;
 
-        private long position = 0;
-        private AtomicLong outstandingDemand = new AtomicLong(0);
-        private boolean writeInProgress = false;
```

Comment:
Why did we have this? lol

Reply:
copy paste error from FileAsyncResponseTransformer I guess, lol